packages/common/src/downloads/_internal/file-url-fetchers/fetchExportImageDownloadFile.ts

```diff
@@ -0,0 +1,109 @@
+import { request } from "@esri/arcgis-rest-request";
+import {
+  DownloadOperationStatus,
+  IFetchDownloadFileOptions,
+  IFetchDownloadFileResponse,
+  ServiceDownloadFormat,
+} from "../../types";
+import HubError from "../../../HubError";
+import { getProp } from "../../../objects/get-prop";
+import { IHubEditableContent } from "../../../core/types/IHubEditableContent";
+import { ExportImageFormat } from "../_types";
+
+/**
+ * @private
+ *
+ * Fetches a download file url from an Image Service via the exportImage endpoint.
+ *
+ * @param options options for refining / filtering the resulting download file
+ * @returns a blob containing the download file
+ */
+export async function fetchExportImageDownloadFile(
+  options: IFetchDownloadFileOptions
+): Promise<IFetchDownloadFileResponse> {
+  validateOptions(options);
+
+  const { entity, format, context, progressCallback } = options;
+  progressCallback && progressCallback(DownloadOperationStatus.PENDING);
+
+  const extent = getExportImageExtent(options);
+  const { xmin, xmax, ymin, ymax } = extent;
+  // TODO: do we need to handle latestWkid as well?
+  const { wkid } = extent.spatialReference;
+
+  const requestOptions = { ...context.requestOptions };
+  requestOptions.httpMethod = "GET";
+  requestOptions.params = {
+    bbox: `${xmin},${ymin},${xmax},${ymax}`,
+    bboxSR: `${wkid}`,
+    f: "image",
+    format,
+    mosaicRule:
+      '{"ascending":true,"mosaicMethod":"esriMosaicNorthwest","mosaicOperation":"MT_FIRST"}',
+  };
+
+  // TODO: Figure out whether we want to leverage the server's maxImageWidth and maxImageHeight.
+  // While it results in higher quality images, it also creates a lot of variability in the result
+  // when we filter an image by extent.
+  //
+  // const { maxImageHeight, maxImageWidth } =
+  //   getProp(entity, "extendedProps.server") || {};
+  // if (maxImageWidth && maxImageHeight) {
+  //   requestOptions.params.size = `${maxImageWidth},${maxImageHeight}`;
+  // }
+
+  const blob: Blob = await request(`${entity.url}/exportImage`, requestOptions);
+  progressCallback && progressCallback(DownloadOperationStatus.COMPLETED);
+  return {
+    type: "blob",
+    blob,
+    filename: getBlobFilename(entity, format as ExportImageFormat),
+  };
+}
+
+function validateOptions(options: IFetchDownloadFileOptions) {
+  const { geometry } = options;
+
+  if (geometry && geometry.type !== "extent") {
+    throw new HubError(
+      "fetchExportImageDownloadFileUrl",
+      "Only extent geometric filters are supported for this type of download"
+    );
+  }
+}
+
+function getExportImageExtent(
+  options: IFetchDownloadFileOptions
+): __esri.Extent {
+  const { entity, geometry } = options;
+  const serverExtent = getProp(entity, "extendedProps.server.extent");
+
+  // TODO: Factor in entity.extent if it exists AND is a valid 4326 bbox
```

Comment:
highlighting for visibility

Reply:
I think we need to bring in someone who uses raster data to get some clarity on what's actually useful in terms of extent. Roll with this, but expect to get some O_o feedback from users

Reply:
Also encapsulated in https://devtopia.esri.com/dc/hub/issues/10902

Reply:
> I think we need to bring in someone who uses raster data to get some clarity on what's actually useful in terms of extent. Roll with this, but expect to get some O_o feedback from users

@dbouwman 100% agree. I think the biggest take away from this ticket has been that we don't understand how image services _really_ work or how our customers intend to use them in the context of hub. We definitely should answer both as time goes on. We can use the follow-on ticket I added above ^^^ as a place of discussion.